admin/model/billmate/payment/bmsetup.php

```diff
@@ -49,4 +59,22 @@ public function unregisterEvents()
         $this->model_setting_event->deleteEventByCode(self::BILLMATE_CHECKOUT_HASH_CODE);
         $this->model_setting_event->deleteEventByCode(self::BILLMATE_PUSH_EVENTS_CODE);
     }
+
+    public function addModifications()
```

Comment:
Add missing PHPdoc

Reply:
@MattiasNilsson I didn't add any PHPDoc, because this method has no arguments and doesn't return anything. If we need to add PHPdoc (just method comment) for all same code parts I would create a separate task to do it for whole module code. How do you think?